userspace/libsinsp/sinsp_filtercheck.cpp

```diff
@@ -947,12 +947,17 @@ bool sinsp_filter_check::compare_rhs(cmpop op,
 
 bool sinsp_filter_check::extract_nocache(sinsp_evt* evt,
                                          std::vector<extract_value_t>& values,
+                                         std::vector<extract_offset_t>* offsets,
                                          bool sanitize_strings) {
 	values.clear();
 	extract_value_t val;
 	val.ptr = extract_single(evt, &val.len, sanitize_strings);
 	if(val.ptr != NULL) {
 		values.push_back(val);
+		if(offsets) {
+			extract_offset_t offset = {1, 0};
+			offsets->push_back(offset);
```

Comment:
```suggestion
			offsets->emplace_back(1, 0);
```
should do the trick, i guess